### PR TITLE
Add Jenkins integration

### PR DESCRIPTION
**Pull request by jawadqur**

### CODE REVIEW

File: docker-compose-ci.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  app:
+    image: quay.io/ncigdc/jenkins-agent:multipython
+    environment:
+      SSH_AUTH_SOCK: $SSH_AUTH_SOCK
+      HTTPS_PROXY: http://cloud-proxy:3128
+      HTTP_PROXY: http://cloud-proxy:3128
+    volumes:
+    - .:/home/jenkins
+    - $SSH_AUTH_SOCK:$SSH_AUTH_SOCK
+    command: bash -c "mkdir -p ~/.ssh && echo -e 'Host github.com\n\tStrictHostKeyChecking no\n' >> ~/.ssh/config && rm -rf .tox && find . \( -name '__pycache__' -or -name '*.pyc' \) -delete  && ./wait-for-it.sh localhost:5432 -t 120 &&  pip install tox -q --user && tox"
```
Comment (anmaxvl) [inline docker-compose-ci.yaml:21]: I think you should be able to do similar thing with `tox -r` like you did in other repos?

Comment (jawadqur) [reply docker-compose-ci.yaml]: 👍 I have updated to only include what is _really_ needed.

File: test/test_gdc_postgres_admin.py [hunk 2]
```diff
@@ -107,6 +109,7 @@ def test_create_double(self):
 
         self.engine.execute('SELECT * from node_case')
 
+    @pytest.mark.skip(reason="Causing race conditions, so skipping for now.")
```
Comment (kulgan) [inline test/test_gdc_postgres_admin.py:112]: I ran into this and the other one a few times, especially when testing locally. I fixed it in another PR, not sure now if that PR should go in first :thinking: or maybe we can just merge this and that PR will re-enable the test

Comment (jawadqur) [reply test/test_gdc_postgres_admin.py:112]: Either way is fine by me.

Comment (kulgan) [reply test/test_gdc_postgres_admin.py:112]: lets get this merged, I can update the other PR

**Pull request merged**